docs/components/toaster.md

```diff
@@ -64,192 +57,202 @@ If a global property conflicts with a component’s own property, the component'
+  >
+    <KongIcon />
+    System
+  </KButton>
+</div>
 
-- `timeoutMilliseconds`
+```ts
+$toaster.open({ title: 'Warning', appearance: 'warning' })
+```
 
-<KButton :disabled="timeLeft <= 3" @click="openNotificationElapse({timeoutMilliseconds: 3000, 'appearance': 'success', 'message': `This toaster has a 3 second timeout`})">{{timeLeft > 3 ? 'Open Toaster' : `Closing in ${timeLeft} seconds` }}</KButton>
+### timeoutMilliseconds
 
-```html
+The default timeout, in milliseconds, is `5000` (5 seconds) however you can change this to by passing the `timeoutMilliseconds` property.
+
+<KButton :disabled="timeLeft <= 3" @click="openNotificationElapse({ timeoutMilliseconds: 3000, appearance: 'success', title: 'Notification', message: `This toaster has a 3 second timeout` })">
+  {{timeLeft > 3 ? 'Open Toaster' : `Closing in ${timeLeft} seconds` }}
+</KButton>
+
+```vue
 <template>
   <KButton @click="openNotification(toasterOptions)">Open Toaster</KButton>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue'
-
-export default defineComponent({
-  data() {
-    return {
-      toasterOptions: {
-        appearance: 'success',
-        timeoutMilliseconds: 3000,
-        message: 'This toaster has a 3 second timeout'
-      },
-    }
-  },
-  methods: {
-    openNotification(options: Record<string, any> | string) {
-      this.$toaster.open(options)
-    }
-  }
-})
+<script setup lang="ts">
+import type { Toast } from '@kong/kongponents'
+
+const toasterOptions: Toast = {
+  appearance: 'success',
+  timeoutMilliseconds: 3000,
+  title: 'Notification',
+  message: 'This toaster has a 3 second timeout'
+}
+
+const openNotification = (options: Toast | string) => {
+  $toaster.open(options)
+}
 </script>
 ```
 
 ## Toaster State
 
 You can view the current state of active toasters by calling `this.$toaster.toasters`. Click the buttons below to watch the state change
 
-<KButton class="success my-button" appearance="primary" @click="openNotification({timeoutMilliseconds: 10000, message: 'Success Notification', appearance: 'success'})">Open Toaster</KButton>
-<KButton appearance="danger" @click="openNotification({'appearance': 'danger', 'message': 'Danger Notification'})" class="my-button">Open Toaster</KButton>
-<KButton @click="openNotification('Basic Notification')">Open Toaster</KButton>
-
-<pre class="language-json">
-<code>
-{{ JSON.stringify(toasters || [], null, 2) }}
-</code>
+<div class="horizontal-container">
+  <KButton @click="openNotification({ timeoutMilliseconds: 10000, title: 'Info Notification', appearance: 'info' })">
+    Open Toaster
+  </KButton>
+  <KButton
+    @click="openNotification({ title: 'Danger Notification', appearance: 'danger' })"
+    appearance="danger"
+  >
+    Open Toaster
+  </KButton>
+  <KButton
+    @click="openNotification('Basic Notification')"
+    appearance="secondary"
+  >
+    Open Toaster
+  </KButton>
+</div>
+
+<pre>
+  {{ JSON.stringify(toasters || [], null, 2) }}
```

Comment:
I like the idea of showing this data, but should the container have a max-height so the page doesn't grow/shrink?

Reply:
Made it fixed height and scrollable